tools/infer/utility.py

```diff
@@ -25,6 +25,8 @@
 import random
 from ppocr.utils.logging import get_logger
 
+import paddle_custom_device.gcu.passes as gcu_passes
```

Comment:
Regular users don't really need `paddle_custom_device`.

Reply:
You are right, this needs to be corrected. Thank you for the correction.